docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -160,6 +160,39 @@ the dependencies as shown below.
     :start-after: [START main_flow]
     :end-before: [END main_flow]
 
+Using the Taskflow API with Docker or Virtual Environments
+----------------------------------------------------------
+
+As of Airflow <Airflow version>, you will have the ability to use the Taskflow API with either a
```

Comment:
TODO: Update the version

Reply:
@jhtimmins this unfortunately can't be done until we know which version we add it to.